dynamiq/types/streaming.py

```diff
@@ -65,12 +65,14 @@ class StreamingConfig(BaseModel):
         timeout (float | None): Timeout for streaming. Defaults to None.
         input_queue (Queue | None): Input queue for streaming. Defaults to None.
         input_queue_done_event (Event | None): Event to signal input queue completion. Defaults to None.
+        is_final (bool): Whether the streaming is final. Defaults to False.
```

Comment:
Sorry, both the name of param and description are not clear.

Reply:
Option for stream all steps or only final version

Reply:
Let's use mode with Enum as you used previously